### PR TITLE
TR3FIR-478 Adding support to setup an already initialized CPU2

### PR DESCRIPTION
**Pull request by TylerGajewski-Token**

### CODE REVIEW

**Review by lukasz-zajac — CHANGES_REQUESTED** [review]
Needs version bump

**Review by dboleyToken — APPROVED** [review]
These changes don't affect normal operation with non SBSFU bootloader. So are safe to pull into current halo develop.

**Comment by lukasz-zajac** [comment]
@TylerGajewski-Token I forgot that we don't use artifactory for embassy anymore, so there is no need to bump anything. My mistake, sorry. You can remove the last commit you've added

File: embassy-usb/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "embassy-usb"
-version = "0.2.0"
+version = "0.2.1"
```
Comment (lukasz-zajac) [inline embassy-usb/Cargo.toml]: If you bumped this only because of my previous review, this should also be removed.

**Review by lukasz-zajac — APPROVED** [review]

**Pull request merged**